### DISCUSSION TITLE
Copilot association CANNOT accept with tab key

### DISCUSSION THREAD

**Discussion by Yang-Kyle** [community/community#20330] [Projects and Issues]
### tab key
I installed the `compilot plug-in` on `VSCode` and when its associative words appeared,
I can't fill in this word with the tab key,but there are a few more spaces!!?

**Comment by FahdSeddik** [comment]
Hey!
I think you might find details in this [link](https://github.com/microsoft/vscode/issues/130674). Hope this helps!

**Reply by Yang-Kyle** [reply]
Thank you. I can use it now. ✌
